lang/en/settings.php

```diff
@@ -21,5 +21,13 @@
 $lang['doorkeeper-secret']   = '(Example) The Secret of your registered Doorkeeper Application.';
 $lang['doorkeeper-authurl']  = '(Example) The authorization endpoint URL of your Doorkeeper setup.';
 $lang['doorkeeper-tokenurl'] = '(Example) The access token endpoint URL of your Doorkeeper setup.';
-$lang['singleService']            = 'Login with single oAuth service only (disables local logins!)';
+$lang['custom-key']          = '(Custom) The Application ID of your registered Doorkeeper Application.';
```

Comment:
Splitbrain,
if your own code development documentation says to reference Doorkeeper, what is the OP meant to do?

`This plugin can easily integrate with your own service. A “Generic” backend class allows for easy configuration of endpoints. However the actual API call for fetching user data still needs to be implemented. The plugin comes with an example Adapter class called “Doorkeeper” which implements oAuth against the demo setup of the Ruby oAuth library Doorkeeper.`